src/Materials/Textures/babylon.texture.ts

```diff
@@ -65,6 +65,24 @@
         @serialize()
         public wAng = 0;
 
+        /**
+         * Defines the center of rotation (U)
+         */
+        @serialize()
+        public uCenter = 0.5;

```

Comment:
Any opinions on the naming of this? It's just for rotation and not for scale. Should I call it uRotationCenter perhaps? It's a bit odd compared to uAng which is also for rotation.

Reply:
I guess uRotationCenter is good.
uAng is OK as you cannot scale with an angle

Reply:
Done!